components/emoji_picker/modules/emoji_tabset.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="d-emoji-picker__tabset">
+    <dt-tab-group
+      tab-list-class="d-emoji-picker__tabset-list"
+      :selected="selectedTab"
+    >
+      <template #tabs>
+        <dt-tab
+          v-for="tab in tabs"
+          :id="tab.id"
+          :key="tab.id"
+          :panel-id="tab.panelId"
+          :label="tab.label"
+          class="d-ml2 d-mr2"
+          @click.capture.stop="selectTabset(tab.id)"
+        >
+          <dt-icon
+            size="400"
+            :name="tab.icon"
+          />
+        </dt-tab>
+      </template>
+    </dt-tab-group>
+  </div>
+</template>
+
+<script setup>
+import DtTabGroup from '@/components/tabs/tab_group.vue';
+import DtTab from '@/components/tabs/tab.vue';
+import DtIcon from '@/components/icon/icon.vue';
+import { computed, ref, toRefs, watch } from 'vue';
+
+const props = defineProps({
+  /**
+   * Whether to show the recently used tab or not
+   * @type {Boolean}
+   * @default false
+   */
+  showRecentlyUsedTab: {
+    type: Boolean,
+    default: false,
+  },
+
+  scrollIntoTab: {
+    type: Number,
+    required: true,
+  },
+
+  isScrollingWithScrollTo: {
```

Comment:
isCurrentlyScrolling?

Reply:
Mhmm, now after the design update, we dont have animation, so... we dont have handleScroll and isScrolling anymore, maybe we can replace `isScrollingWithScrollTo` to `isScrolling` , what you think

Reply:
yep sounds good